variables.tf

```diff
@@ -18,6 +18,18 @@ variable "create_sns_topic" {
   default = false
 }
 
+variable "sns_encryption_enabled" {
+  description = "Whether to enable encryption for the SNS topic. Default `false`"
+  type        = bool
+  default     = false
+}
+
+variable "sns_encryption_key_id" {
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK."
+  type        = string
+  default     = "alias/aws/sns"
```

Comment:
Hi Kevin! Thanks for this PR!

Did you test the delivery with this default SNS key? From my experience, it is often the case when these default AWS KMS keys do not have AWS own principals allowed, and so cannot be used for SNS notifications from the AWS side.

Reply:
@max-lobur the `sns_encryption_key_id` is fed into `kms_master_key_id` which has the same default value

https://github.com/cloudposse/terraform-aws-sns-topic#input_kms_master_key_id

Reply:
That's right, but it doesn't mean it will work in your case :) 

`terraform-aws-sns-topic` is a standalone SNS module where default KMS will work in most cases. In your case there's always going to be only one producer in that queue, which is AWS principal, which I have doubts will work with this KMS key 

cc @nitrocode

Reply:
Sorry for delay @kevcube @nitrocode 
I have verified the proposed default configuration here works as desired, AWS principal is whitelisted there!